apps/vscode-wing/.projenrc.ts

```diff
@@ -105,6 +105,38 @@ const contributes: VSCodeExtensionContributions = {
       },
     },
   ],
+  commands: [
```

Comment:
Can we make this open as a pane instead of a tab? (like the markdown preview)

Reply:
We can, but I don't think we should yet because the console is kinda unusable in a split view on anything other than an ultrawide screen. I think we need to work on the responsive styling first or something to help it be usable.

A user can move it into a split pane if they wish though.

Reply:
+1 that we should work on the responsive styling.

Is it possible to check the screen width and decide if we open a tab or a split view based on that?

When I work with the Console, it's extremely useful to be able to see both the code and the console at the same time.

Reply:
> Is it possible to check the screen width and decide if we open a tab or a split view based on that?

I can't find an API for it. Generally, vscode prevents raw access to the DOM so if there's no API that may not be information we can find.

> When I work with the Console, it's extremely useful to be able to see both the code and the console at the same time.

Totally agree, so I'll try it out and split it by default. Hopefully most people have big enough screens until the responsiveness improves.